dashboard/test/controllers/api/v1/sections_controller_test.rb

```diff
@@ -1077,39 +1077,63 @@ def returned_section
 
   test 'can get all code review groups for a section' do
     sign_in @teacher
-
-    # create a new section to avoid extra unassigned students
-    code_review_group_section = create(:section, user: @teacher, login_type: 'word')
-    # Create 5 students
-    followers = []
-    5.times do |i|
-      student = create(:student, name: "student_#{i}")
-      followers << create(:follower, section: code_review_group_section, student_user: student)
-    end
-
-    # Create 2 code review groups
-    group1 = CodeReviewGroup.create(section_id: code_review_group_section.id, name: "group1")
-    group2 = CodeReviewGroup.create(section_id: code_review_group_section.id, name: "group2")
-    # put student 0 and 1 in group 1, and student 2 in group 2
-    CodeReviewGroupMember.create(follower_id: followers[0].id, code_review_group_id: group1.id)
-    CodeReviewGroupMember.create(follower_id: followers[1].id, code_review_group_id: group1.id)
-    CodeReviewGroupMember.create(follower_id: followers[2].id, code_review_group_id: group2.id)
-
-    get :code_review_groups, params: {id: code_review_group_section.id}
-    group1_members = [{follower_id: followers[0].id, name: followers[0].student_user.name}, {follower_id: followers[1].id, name: followers[1].student_user.name}]
-    group2_members = [{follower_id: followers[2].id, name: followers[2].student_user.name}]
+    set_up_code_review_groups
+    get :code_review_groups, params: {id: @code_review_group_section.id}
+    group1_members = [
+      {follower_id: @followers[0].id, name: @followers[0].student_user.name},
+      {follower_id: @followers[1].id, name: @followers[1].student_user.name}
+    ]
+    group2_members = [{follower_id: @followers[2].id, name: @followers[2].student_user.name}]
     unassigned_members = [
-      {follower_id: followers[3].id, name: followers[3].student_user.name},
-      {follower_id: followers[4].id, name: followers[4].student_user.name}
+      {follower_id: @followers[3].id, name: @followers[3].student_user.name},
+      {follower_id: @followers[4].id, name: @followers[4].student_user.name}
     ]
     expected_response = {
       groups: [
-        {id: group1.id, name: group1.name, members: group1_members},
-        {id: group2.id, name: group2.name, members: group2_members},
+        {id: @group1.id, name: @group1.name, members: group1_members},
+        {id: @group2.id, name: @group2.name, members: group2_members},
         {unassigned: true, members: unassigned_members}
       ]
     }
     assert_response :success
     assert_equal(expected_response.as_json, json_response)
   end
+
+  test 'post code_review_groups returns 400 for invalid group' do
```

Comment:
Do we have a positive test for the new POST route?

Reply:
I didn't have one here because I have the logic tested in the model, but I'll add a basic one that tests for success.